packages/core/src/getDefaultRegistry.ts

```diff
@@ -1,15 +1,20 @@
 import { Registry, RegistryFieldsType, RegistryWidgetsType } from "@rjsf/utils";
 
 import fields from "./components/fields";
+import templates from "./components/templates";
 import widgets from "./components/widgets";
 
-/**  */
-export default function getDefaultRegistry<T = any, F = any>(): Partial<
-  Registry<T, F>
+/** The default registry consists of all the fields, templates and widgets provided in the core implementation,
+ * plus an empty `rootSchema` and `formContext.
```

Comment:
Add "We omit `schemaUtils` here because it cannot be defaulted without a `rootSchema` and `validator`. It will be added into the computed registry later in the `Form`.